Dockerfile

```diff
@@ -7,4 +7,4 @@ VOLUME ${APP_DIR}
 
 WORKDIR ${APP_DIR}
 
-CMD npm i && npm start
+CMD npm install webpack-dev-server && npm i && npm start
```

Comment:
I don't think it's going to help with missing git issue and `webpack-dev-server` is already in package.json. What exactly are you trying to do here?